src/components/Form.jsx

```diff
@@ -31,12 +31,12 @@ const Form = () => {
   };
 
   return (
-    <form action="">
-      <h2>ADD NEW BOOK</h2>
-      <fieldset>
-        <input type="text" name="title" placeholder="Book title" className=" border" value={formData.title} onChange={handleInputChange} />
-        <input type="text" name="author" placeholder="author" className=" border" value={formData.author} onChange={handleInputChange} />
-        <button type="submit" className=" bg-blue-500 p-2 text-white" onClick={handleSubmit}>ADD BOOK</button>
+    <form action="" className=" border-t mt-8 py-3 md:w-[90%] mx-auto">
+      <h2 className=" font-bold text-gray-500 py-3">ADD NEW BOOK</h2>
+      <fieldset className="grid md:grid-cols-[3fr,1fr,.5fr] gap-2">
+        <input type="text" name="title" placeholder="Book title" className=" border rounded-md px-4 py-2" value={formData.title} onChange={handleInputChange} />
+        <input type="text" name="author" placeholder="author" className=" border rounded-md px-4 py-2" value={formData.author} onChange={handleInputChange} />
+        <button type="submit" className=" bg-[rgb(2,144,255)] p-2 text-white rounded-md mt-4 md:mt-0" onClick={handleSubmit}>ADD BOOK</button>
```

Comment:
- [ ] The "Add Book" button should have a `Roboto Slab` font family as shown below so kindly add this as well.
- Original design:
  ![image](https://github.com/Akorede3133/bookstore/assets/104968236/100ca0d1-6942-425c-ba39-06fdeb78870f)
- Your design:
  ![image](https://github.com/Akorede3133/bookstore/assets/104968236/4af6fa73-da4e-4088-ada7-2e3cd17447e7)